### ISSUE TITLE
Will there be more releases to crates.io?

### ISSUE THREAD

**Issue by atrakh** [arlyon/async-stripe#318]
### Is your feature request related to a problem? Please describe.

Hi there, I noticed the repository on crates.io for this package hasn't been updated in a while, while the code in git has -- are there plans to continue releasing new version to crates.io?

**Comment by arlyon** [comment]
Hi! I there are a few things open issues I've been meaning to fix for a good while now but life has been busy recently. I will see if I can get something like semantic release automations working so that new releases happen automatically when PRs are merged. Thank you for the suggestion!

**Comment by arlyon** [comment]
I actually released a new version yesterday, `0.15.1`. I'm _pretty sure_ that automated releases are set up too(just have one more thing that I am testing in this PR https://github.com/arlyon/async-stripe/pull/324).

So you should be able to use

```toml
async-stripe = {version = "0.15", default-features = false, features = ["runtime-tokio-hyper", "checkout", "webhook-events", "connect"] }
```

Let me know if that is not the case :)

**Comment by atrakh** [comment]
Awesome, thank you!
